Zend/zend_execute_API.c

```diff
@@ -181,6 +182,8 @@ void init_executor(void) /* {{{ */
 	EG(each_deprecation_thrown) = 0;
 
 	EG(active) = 1;
+
+	zend_weakrefs_init();
```

Comment:
Not that it matters, but it would be more obvious to move this prior to active=1.